src/__experimental__/components/date-range/date-range.component.js

```diff
@@ -22,12 +22,22 @@ class DateRange extends React.Component {
 
   state = {
     startDateValue: {
-      formattedValue: DateHelper.formatDateToCurrentLocale(this.startDate),
-      rawValue: DateHelper.formatValue(this.startDate || this.today),
+      formattedValue: this.props.startDateProps?.allowEmptyValue
+        ? ""
+        : DateHelper.formatDateToCurrentLocale(this.startDate),
+      rawValue: DateHelper.formatValue(
+        this.startDate ||
+          (this.props.startDateProps?.allowEmptyValue ? "" : this.today)
```

Comment:
Nice, I wasn't aware that optional chaining works on our config :)

Reply:
It's turned on by default in ``preset-env`` now (since 7.8) ;)